### DISCUSSION TITLE
Is it possible to reference another models field?

### DISCUSSION THREAD

**Discussion by miki-tebe** [ComfortablyCoding/strapi-plugin-slugify#44] [Q&A] [answered]
For example there are two models Article and Blog, Blog has a title field and Article has a slug field and is it possible to slugify Articles slug field with Blogs title field? 
something like 
`
article: {
          field: "slug",
          references: {
            blog: {
              field: "title",
            },
          },
        },
`

**Comment by ComfortablyCoding** [comment, marked as answer]
No, that is not possible at the moment. I also do not see this as a common usecase.
